src/main/java/com/gamegoo/gamegoo_v2/content/board/controller/BoardController.java

```diff
@@ -35,4 +41,35 @@ public ApiResponse<BoardInsertResponse> boardInsert(
         return ApiResponse.ok(boardFacadeService.createBoard(request, member));
     }
 
+    /**
+     * 게시글 목록 조회 API
+     */
+    @GetMapping("/list")
+    @Operation(summary = "게시판 글 목록 조회 API",
+            description = "게시판 글 목록을 조회하는 API 입니다. 필터링을 원하면 각 파라미터를 입력하세요.")
+    @Parameters({
+            @Parameter(name = "pageIdx", description = "조회할 페이지 번호를 입력해주세요. 페이지 당 20개의 게시물을 볼 수 있습니다."),
+            @Parameter(name = "mode", description = "(선택) 게임 모드를 입력해주세요. < 빠른대전: 1, 솔로랭크: 2, 자유랭크: 3, 칼바람 나락: 4 >"),
+            @Parameter(name = "tier", description = "(선택) 티어를 선택해주세요."),
+            @Parameter(name = "mainPosition", description = "(선택) 포지션을 입력해주세요. < 전체: 0, 탑: 1, 정글: 2, 미드: 3, 바텀: 4, " +
+                    "서포터: 5 >"),
+            @Parameter(name = "mike", description = "(선택) 마이크 여부를 선택해주세요.")
+    })
+    public ApiResponse<BoardResponse> boardList(
+            @RequestParam(defaultValue = "1") int pageIdx,
+            @RequestParam(required = false) Integer mode,
+            @RequestParam(required = false) Tier tier,
+            @RequestParam(required = false) Integer mainPosition,
+            @RequestParam(required = false) Boolean mike) {
+        // <포지션 정보> 전체: 0, 탑: 1, 정글: 2, 미드: 3, 바텀: 4, 서포터: 5
+        if (mainPosition != null && mainPosition == 0) {
+            // 전체 포지션 선택 시 필터링에서 제외
+            mainPosition = null;
+        }
+
+        BoardResponse response = boardFacadeService.getBoardList(mode, tier, mainPosition, mike, pageIdx);
+        return ApiResponse.ok(response);
+    }
```

Comment:
이 부분을 
```suggestion
return ApiResponse.ok(boardFacadeService.getBoardList(mode, tier, mainPosition, mike, pageIdx));
```
으로 변경해도 좋을 것 같아요!